src/plugins/xerces/README.md

```diff
@@ -2,10 +2,11 @@
 - infos/author = e1528532 <e1528532@libelektra.org>
 - infos/licence = BSD
 - infos/provides = storage/xml
+- infos/needs =
 - infos/placements = getstorage setstorage
 - infos/status = recommended experimental unittest
+- infos/metadata = elektraXercesOriginalRootName
```

Comment:
Yeah, there is a more general issue of directly exposing meta data within a configuration file involved here. (The same issue exists with any meta data., e.g. comments)

We would need a way to specify if attributes are meta data to be interpreted by Elektra. But this is something for the next PR!

Btw. github cuts away XML.